docs/NAVIGATION.md

```diff
@@ -8,6 +8,7 @@ See https://oprypin.github.io/mkdocs-literate-nav/
 - [How-to guides](how-to-guides/index.md)
     - [Basic](how-to-guides/get-citations-basic.py)
     - [Advanced](how-to-guides/get-citations-advanced.py)
+- [Tutorials](tutorials/index.md)
```

Comment:
I know this is empty, but it helps provide structure to the docs and makes clear that we don't really have tutorials yet